src/sentry/models/debugfile.py

```diff
@@ -361,6 +362,23 @@ def _analyze_progard_filename(filename: str) -> Optional[str]:
         return None
 
 
+@region_silo_only_model
+class ProguardArtifactRelease(Model):  # type: ignore
+    __include_in_export__ = False
+
+    organization_id = BoundedBigIntegerField(db_index=True)
+    project_id = BoundedBigIntegerField(db_index=True)
+    release_name = models.CharField(max_length=250)
```

Comment:
Why do we use the release name here vs a link to the actual release?

Reply:
This is wanted, the goal here, like with sourcemaps, is to have a weak release association. With this association, you will optimistically tell the user to specify a future release that will be created later on down the pipeline. Ofc this won't result in any database consistency mechanisms helping us to maintain data consistency but we aligned on such an approach.